### ISSUE TITLE
disallow merge commits for pull requests 

### ISSUE THREAD

**Issue by MichaelKreil** [shortbread-tiles/shortbread-tilemaker#39]
Merging pull requests with a single commit using an extra merge commit creates a mess of branches:
<img width="470" alt="image" src="https://github.com/user-attachments/assets/98da7e6d-08fb-44cd-9697-17119d9dc914">

I want to change the pull request settings for this repo to disallow merge commits:
<img width="338" alt="image" src="https://github.com/user-attachments/assets/af0e03bc-986c-4858-9d76-e4bba1bd9714">

Ok?

**Comment by Nakaner** [comment]
I agree and unchecked the first checkbox "Allow merge commits" right now.
